### PR TITLE
Add more MSI REST documentation

### PR DESCRIPTION
**Pull request by keharper**
## This PR is a:

- [x] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will provide documentation for MSI REST endpoints that perform the following functions:

* Manage low-quantity notifications
* Check salable quantities for a product
* Manage source selection algorithms

whatsnew
Added documentation for Inventory Management REST endpoints: [low stock notifications](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/manage-low-quantity.html), [salable quantities](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/check-salable-quantity.html), and [source selection algorithms](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/manage-source-selection.html)

### CODE REVIEW

File: guides/v2.3/rest/modules/inventory/check-salable-quantity.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+**Path parameters**
+
+Name | Description | Type
+--- | --- | ---
+`sku` | The SKU of the product | String
+`stock_id` | The ID of the stock the product is assigned to | Integer
+`requestedQty` | The requested quantity of products | Integer
+`type` | The type of sales channel the stock is assigned to. For 2.3, this value must be `website`. | String
+`code` | The name of the sales channel (website) | String
```
Comment (lorikrell) [inline guides/v2.3/rest/modules/inventory/check-salable-quantity.md]: Isn't it the code of the sales channel, not the name?

File: guides/v2.3/rest/modules/inventory/manage-low-quantity.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+group: rest-api
+title: Manage low-quantity notifications
+---
+
+The Admin allows the merchant to configure low-quantity notifications from several locations:
+
+* The **Notify for Quantity Below** field (**Stores** > **Configuration** > **Catalog** > **Inventory** > **Product Stock Options**) sets the default value for all products for the entire website/store.
+* The Advanced Inventory **Notify for Quantity Below** field (**Catalog** > **Products** > specific product > **Advanced Inventory**) overrides the value set at the website/store level. The value applies to all of the product's sources. 
+* The **Notify Quantity** fields (**Catalog** > **Products** > specific product > **Assigned Sources** section) override all other settings. The merchant can assign a different threshold for each source for the specific product. Magento deducts these amounts from the available quantity to provide the total salable quantity.
```
Comment (lorikrell) [inline guides/v2.3/rest/modules/inventory/manage-low-quantity.md]: Maybe move the last sentence up or after the bulleted list to describe the Notify for Quantity Below? Possible revision to consider: "Magento deducts this amount from the total salable quantity for the stock." 
Basically, the amount is deducted from the aggregated amount for all sources assigned to a stock. Side note, unsure you want this here too, if backorders is enabled, this value is not deducted from Salable Quantity.

File: guides/v2.3/rest/modules/inventory/manage-source-selection.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+---
+group: rest-api
+title: Manage source selection algorithms
+---
+
+Inventory Management uses the Source Selection Algorithm (SSA) to track the salable quantities of product inventory across all sources and make recommendations for partial and full shipments. 
+
+The SSA uses stocks and sources to check the sales channel for incoming product requests and determines the available inventory. It
```
Comment (lorikrell) [inline guides/v2.3/rest/modules/inventory/manage-source-selection.md]: Is "It" meant to be there? I know it means the SSA does the following, hmm maybe: The algorithm:

File: guides/v2.3/rest/modules/inventory/manage-source-selection.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+* Calculates the aggregated virtual salable quantity of all assigned sources per stock
+* Subtracts the Out-of-Stock Threshold amount from salable quantity to protect against overselling and support concurrent checkout
+* Reserves inventory quantities at checkout, deducting from in-stock inventory at order processing and shipment
+* Supports backorders with enhanced options for negative thresholds
+
+The SSA also manages shipments by providing recommendations for the best sources to ship partial or all products or override the selections to:
+
+* Ship partial shipments, sending only a few products from specific locations and completing the full order at a later date.
```
Comment (lorikrell) [inline guides/v2.3/rest/modules/inventory/manage-source-selection.md]: Maybe change location to source? Just to keep it the same across all entries, etc?

File: guides/v2.3/rest/modules/inventory/check-salable-quantity.md (hunk 1 above)
Comment (keharper) [reply guides/v2.3/rest/modules/inventory/check-salable-quantity.md]: Yes. Corrected.

**Comment by lorikrell** [comment]
Awesome, @keharper ! Fire when ready.

**Pull request merged**